packages/polka/src/index.ts

```diff
@@ -58,6 +59,7 @@ const methods = [
 interface Request extends p.Request {
   session?: unknown;
   body?: unknown;
+  traceId?: string;
```

Comment:
We can make this `traceId: string`. If you create the traceId inside the handler function, traceId will be always defined.

Reply:
On second thought. We should just use the `requestCtx` for this. The session and body were typed here because polka didn't do it.

Reply:
So what are you recommending I do? Bare in mind, I'm new to Polka and still pretty new to node :D

Reply:
Before I recommend. Could you explain a bit your thinking here?

Reply:
I was just adding the traceId - honestly not a ton of thought on it. I'm just trying to find all the places that traceId needs to go out